apps/images/api/schema.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import graphene
+
+from apps.images.api.types import ImageType
+from apps.images.models import Image
+
+
+class Query(graphene.ObjectType):
+    """GraphQL query for all images."""
+
+    images = graphene.List(ImageType)
+    image = graphene.Field(ImageType, slug=graphene.String())
+
+    def resolve_images(self, info, **kwargs):
```

Comment:
Should we use `parent` instead of `self` on our `resolve_*` methods?
https://docs.graphene-python.org/en/latest/types/objecttypes/#implicit-staticmethod

Reply:
It looks like in the graphene-django docs (e.g., https://docs.graphene-python.org/projects/django/en/latest/queries/), they sometimes name the parameter `root` and sometimes `self`. It won't affect the behavior, but do you think it would be more readable with `root` (and/or the `@staticmethod` decorator)?

Reply:
I don't know enough atm to have an opinion about whether `root` or `parent` would be better. I don't like using `@staticmethod` if it does not change behavior, but it would make it clear that we don't have access to `self`. Mostly, I just don't like the variable being named `self` if it is not `self`.